docs/design-proposals/arch/daemonset.md

```diff
@@ -0,0 +1,31 @@
+## DaemonSet in Arktos
+
+Due to the nature Arktos being a multiple-tenanted system, DaemonSet is only allowed in system tenant; only cluster admin is permitted to create/delete DaemonSet resources.
+
+### DaemonSet in Scale-up Arktos
+Scale-up Arktos has single master. With the admission controller blocking non-system tenant DaemonSet enabled by API Server, Arktos is able to ensure DaemonSet proper support.
+
+### DaemonSet in Scale-out Arktos
+[Scale-out Arktos](arktos_scale_out.md) may have multiple tenant partitions and resource managers; each of them has Kubernetes master components including API Server, controller manager, etc. DaemonSet resource will be managed by tenant partition, as the controllers in tenant partition have the global view of Node resources.
```

Comment:
probably it makes the naming consistent by using "resource partitions" instead of "resource managers"
also, Arktos should be used instead of Kubernetes in the docs.

Reply:
changed as suggested